src/jit/hwintrinsicxarch.h

```diff
@@ -1,306 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The `hwintrinsicxarch.h` and `hwintrinsicarm64.h` files aren't needed anymore, as all of this is being shared now.